.github/workflows/build.yml

```diff
@@ -35,13 +55,17 @@ jobs:
           profile: minimal
       - name: Build on Rust ${{ matrix.toolchain }} with net-tokio
         if: matrix.build-net-tokio
-        run: RUSTFLAGS="-C link-dead-code" cargo build --verbose --color always
+        run: cargo build --verbose --color always
+        env:
+          RUSTFLAGS: "-C link-dead-code"
```

Comment:
We shouldnt try to generate coverage except on Linux, which is the only reason we'd need to link dead code.

Reply:
Oh okay. Perhaps this env var should only be set on 1.39.0 then.